app/admin/RTMediaFormHandler.php

```diff
@@ -170,7 +170,7 @@ public static function radio( $args ) {
 		foreach ( $args['radios'] as $value => $key ) {
 			$args['rtForm_options'][] = array(
 				$key      => $value,
-				'checked' => ( (int) $args['default'] === $value ) ? true : false,
+				'checked' => ( $args['default'] == $value ) ? true : false, // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison The == error ignored as the privacy levels indexes are integer, and while saving in database, the saved value becomes string.
```

Comment:
```suggestion
				'checked' => ( $args['default'] == $value ) ? true : false, // phpcs:ignore WordPress.PHP.StrictComparisons.LooseComparison -- The == error ignored as the privacy levels indexes are integer, and while saving in database, the saved value becomes string.
```

Did you test this by running PHPCS? It'll give error for loose comparison in your current code. `// phpcs:ignore` won't work like this. Please test before committing the code.

Reply:
Added `--` in phpcs ignore comment.